include/vkgcDefs.h

```diff
@@ -601,9 +601,8 @@ struct SamplerYCbCrConversionMetaData {
       unsigned yChromaOffset : 1; ///< COSITED_EVEN(0) or MIDPOINT(1)
       unsigned xSubSampled : 1;   ///< true(1) or false(0)
       unsigned ySubSampled : 1;   ///< true(1) or false(0)
-      unsigned tileOptimal : 1;   ///< true(1) or false(0)
       unsigned dstSelXYZW : 12;   ///< dst selection Swizzle
-      unsigned undefined : 11;
+      unsigned undefined : 12;
```

Comment:
I think your problem is to make compatibility with old codes. So please see VKGC header. A new major version is required and your codes will be
```
#if LLPC_CLIENT_INTERFACE_MAJOR_VERSION >= X
  unsigned : 1; ///< Unused
#else
  unsigned tileOptimal : 1;   ///< true(1) or false(0)
#endif 
```
All your codes should be gated with version check.

Reply:
You are right - I changed this in next uploads.